File: launchdarkly/typescript-node/src/GetBigSegmentExportExample.ts
import * as fs from 'fs';
import api from "launchdarkly_client"
import models from "launchdarkly_client"

const apiCaller = new api.SegmentsApi();
apiCaller.setApiKey(api.SegmentsApiApiKeys.ApiKey, "YOUR_API_KEY");

apiCaller.getBigSegmentExport(
  "projectKey_string", // projectKey
  "environmentKey_string", // environmentKey
  "segmentKey_string", // segmentKey
  "exportID_string", // exportID
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling SegmentsApi#getBigSegmentExport:");
  console.log(error.body);
});

File: launchdarkly/kotlin/src/main/kotlin/oseg/launchdarkly_examples/GetBigSegmentExportExample.kt
package oseg.launchdarkly_examples

import com.launchdarkly.client.infrastructure.*
import com.launchdarkly.client.apis.*
import com.launchdarkly.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class GetBigSegmentExportExample
{
    fun getBigSegmentExport()
    {
        ApiClient.apiKey["Authorization"] = "YOUR_API_KEY"

        try
        {
            val response = SegmentsApi().getBigSegmentExport(
                projectKey = "projectKey_string",
                environmentKey = "environmentKey_string",
                segmentKey = "segmentKey_string",
                exportID = "exportID_string",
            )

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling SegmentsApi#getBigSegmentExport")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling SegmentsApi#getBigSegmentExport")
            e.printStackTrace()
        }
    }
}
